internal/tiltfile/tiltfile.go

```diff
@@ -363,7 +363,7 @@ func (tiltfile Tiltfile) GetManifestConfigs(manifestName string) ([]model.Manife
 	}
 }
 
-func skylarkManifestToDomain(manifest k8sManifest, repos []gitRepo) (model.Manifest, error) {
+func skylarkManifestToDomain(manifest k8sManifest, repos map[string][]gitRepo, name model.ManifestName) (model.Manifest, error) {
```

Comment:
this seems...weird

would it be easier if the git repo was attached to the k8sManifest object? i don't understand why it's not.

Reply:
yeah, let's see what refactors I can do here.

Reply:
on second thought, this may be totally redundant? i'm pretty sure you can pull the git repo off of manifest.mount.localPath.repo.

Reply:
This doesn't work out of the box but maybe that's a simpler way to associate repos with mounts in the tiltfile. Let me play around with that.